shell/app/styles/antd-extension.scss

```diff
@@ -417,6 +417,14 @@ div.ant-table {
   }
 }
 
+.ant-table .ant-form-item-has-error {
+  margin: 10px 0 !important;
+
+  .ant-form-item-explain-error {
+    font-size: 12px;
+  }
+}
+
```

Comment:
Why add this overwrite style?
![image](https://user-images.githubusercontent.com/3955437/128328822-d629c571-318a-416e-9916-db69a53e7991.png)